.buildkite/pipeline.yml

```diff
@@ -8,6 +8,33 @@ env:
   IMAGE_ID: xcode-15.4
 
 steps:
+  - label: Windows test inline Bash
+    command: |
+      echo "I'm about to exit 2. I expect this step to fail..."
+      exit 2
+    agents:
+      queue: windows
+
+  - label: Windows test Bash script
+    command: .buildkite/commands/command-that-fails.sh
+    agents:
+      queue: windows
+
+  - label: Windows test Bash script no set -e
+    command: .buildkite/commands/command-that-fails-without-set-e.sh
+    agents:
+      queue: windows
+
+  - label: Windows test PS1 script that fails
+    command: .buildkite/commands/command-that-fails-internally.ps1
+    agents:
+      queue: windows
```

Comment:
Fails as expected in https://buildkite.com/automattic/ci-toolkit-buildkite-plugin/builds/1217

Meaning: Calling `Exit [>0]` from a PS1 script result in the CI step failing.